userguide/tutorials/what_is_kill_bill.adoc

```diff
@@ -2,26 +2,30 @@
+Similar to the Unix philosophy of composability over monolithic design, Kill Bill provides APIs to create various pricing models, manage subscriptions, generate invoices, integrate with various payment gateways, etc. Whether you are selling access to a digital service for $10 a month or selling a multi-tier cell-phone data plan, Kill Bill provides the primitives to do it so you don't have to start from scratch.
 
-The *payment* features let you connect with virtually any payment gateway and processor (Stripe, Braintree, PayPal, Adyen, CyberSource, etc.) to charge your customers (using credit or debit cards, ACH, wire transfers, Bitcoin, etc.) for their subscriptions or one-off payments (e.g. shopping cart-type websites like Amazon or Zappos).
+_Does Kill Bill support per CPU-hour billing?_ is the wrong kind of question to ask when evaluating Kill Bill. This would be similar to asking _Does Spring Boot offer endpoints to store user data?_: Spring Boot gives you all the tools you need to create endpoints which can take in user data and let you persist that data in a database, without having to worry about parsing HTTP headers or setting up database connection pooling, but it doesn't offer a user data endpoint. This is similar in the case of Kill Bill: while it doesn't understand _CPU hour billing_, you can create a usage-based plan in the catalog, which is billed based on the number of consumed _CPU-hour_ units.
 
+While Kill Bill is used for various cases (b2b, b2c, offline, etc.), to get a better sense of what Kill Bill offers out of the box, the easiest is to look at how it is used in a typical b2c e-commerce environment.
 
-== Overview
+The *subscription billing* features allow an online store to create recurring or usage based subscriptions (e.g. "gold plan subscription at $9.99/month") and manage upgrades and downgrades. The system will take care of generating invoices, managing credits and triggering payments. Netflix, Dropbox and The New York Times are examples of websites which sell such subscriptions.
+
+The *payment* features let the store connect with virtually any payment gateway and processor (Stripe, Braintree, PayPal, Adyen, etc.) to charge your customers (using credit or debit cards, ACH, wire transfers, Bitcoin, etc.) for their subscriptions or one-off payments (e.g. shopping cart-type websites like Amazon or Zappos).
 
 Check this video for a quick tour:
 
 https://www.youtube.com/watch?v=f2IHcz3OLYo[image:http://img.youtube.com/vi/f2IHcz3OLYo/0.jpg[align=center]]
 
+== Overview
+
 === Big Picture
 
-At a high level we provide 2 pieces of software that **you will have to deploy and manage** in your own data center or in any public cloud.
-//TODO For convenience we also offer a managed version running in AWS that you can use for free to get started very quickly.
+At a high level, we provide 2 pieces of software that **you will have to deploy and manage** in your own data center, in your private cloud, or in any public cloud.
```

Comment:
```suggestion
At a high level, we provide two pieces of software that **you will have to deploy and manage** in your own data center, in your private cloud, or in any public cloud.
```

Reply:
Not sure about rules either, but the arrowheads show the flow of HTTP traffic (requests). I think it makes sense to keep them.

You do have a point regarding the other high level graphic missing a key, but I'm not sure what I would put as a legend (it's logically external / internal traffic, but I think calling it that way would be confusing...).